### ISSUE TITLE
Settings screen enhancement

### ISSUE THREAD

**Issue by TeChn4K** [adblockradio/webplayer#10]
I throw some ideas here:
- Settings should not prevent user from exiting to main screen.
Actually, you have to go to the end of the tunnel to be able to exit. IMO a modal box could be better, closable at any time.
- Settings should be saved instantly
- We should be able to jump to any page from the breadcrumb.

Any other ideas?

**Comment by TeChn4K** [comment]
We have to keep the initial settings tunnel in mind (first visit): users should not be able to start listening before. All settings must be defined at least once.

**Comment by dest4** [comment]
> We should be able to jump to any page from the breadcrumb
It's already the case, at least backwards. We could make it also available forward, if it is relevant (e.g. at least a radio has been added in the playlist)

**Comment by dest4** [comment]
> modal box could be better, closable at any time

Good idea, but you should pay attention to the mobile interface, where space is constrained.
